packages/openapi-v3/src/generate-schema.ts

```diff
@@ -10,7 +10,7 @@ import {SchemaObject} from '@loopback/openapi-v3-types';
  * parameter's type.
  * `type` and `format` will be preserved if provided in `schema`
  *
- * @private
+ * @internal
```

Comment:
Does `strong-docs` recognize `@internal` annotation? If not, then I think this change is going to break API docs that we are generating right now :(